jquery.flot.legend.js

```diff
@@ -3,14 +3,14 @@
 */
 
 (function($) {
-    var placeholder, options;
+    var placeholder;
 
     var options = {
         legend: {
             show: true,
-            noColumns: 1, // number of colums in legend table
-            labelFormatter: null, // fn: string -> string
-            labelBoxBorderColor: "#ccc", // border color for the little label boxes
+            // noColumns: 1, // number of colums in legend table
+            // labelFormatter: null, // fn: string -> string
+            // labelBoxBorderColor: "#ccc", // border color for the little label boxes
```

Comment:
If they are not used then feel free to remove them. The same for any unused piece of code.